api/docs/release.dox

```diff
@@ -148,6 +148,8 @@ Further non-compatibility-affecting changes include:
  - Added drmgr_register_opcode_instrumentation_event() and
    drmgr_unregister_opcode_instrumentation_event() so that drmgr supports
    opcode event instrumentation.
+ - Added dr_get_app_args() and dr_app_arg_as_utf8()  to enable the gathering
+   of application command-line args on Unix systems.
```

Comment:
Not just UNIX

Reply:
OK it's NYI on Windows -- maybe reword to say it's only currently implemented for UNIX

Reply:
Done.